src/patternfly/demos/Card/examples/Card.md

```diff
@@ -1006,7 +1006,7 @@ import './Card.css'
             Readiness probe failed: Get https://10.131.0.7:5000/healthz: dial tcp 10.131.0.7:5000: connect: connection refused
           {{/description-list-description}}
           {{#> description-list-description}}
-            <time class="pf-v6-u-color-200 pf-v6-u-font-size-sm">Jun 17, 11:02 am</time>
+            <time class="pf-v6-u-text-color-subtle pf-v6-u-font-size-sm">Jun 17, 11:02 am</time>
```

Comment:
The [timestamp component](https://www.patternfly.org/components/timestamp) looks like it would be perfect here